src/librbd/cache/pwl/rwl/WriteLog.cc

```diff
@@ -882,60 +938,27 @@ void WriteLog<I>::flush_pmem_buffer(V& ops)
 template <typename I>
 void WriteLog<I>::persist_last_flushed_sync_gen()
 {
-  TOID(struct WriteLogPoolRoot) pool_root;
-  pool_root = POBJ_ROOT(m_log_pool, struct WriteLogPoolRoot);
-  uint64_t flushed_sync_gen;
-
   std::lock_guard append_locker(this->m_log_append_lock);
-  {
-    std::lock_guard locker(m_lock);
-    flushed_sync_gen = this->m_flushed_sync_gen;
-  }
-
-  if (D_RO(pool_root)->flushed_sync_gen < flushed_sync_gen) {
-    ldout(m_image_ctx.cct, 15) << "flushed_sync_gen in log updated from "
-                               << D_RO(pool_root)->flushed_sync_gen << " to "
-                               << flushed_sync_gen << dendl;
-    TX_BEGIN(m_log_pool) {
-      D_RW(pool_root)->flushed_sync_gen = flushed_sync_gen;
-    } TX_ONCOMMIT {
-    } TX_ONABORT {
-      lderr(m_image_ctx.cct) << "failed to commit update of flushed sync point" << dendl;
-      ceph_assert(false);
-    } TX_FINALLY {
-    } TX_END;
-  }
+  persist_pmem_root();
 }
 
 template <typename I>
 void WriteLog<I>::reserve_cache(C_BlockIORequestT *req,
```

Comment:
I think we can remove this function.  New allocator don't need this like SSD allocator.